<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FF7F50</string>
  </property>
  <widget class="QLabel" name="RegistrLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Придумайте пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Придумайте логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Код подтверждения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finishRegButt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> display: inline-block;
    text-decoration: none;
    background-color: #ffa852;
    color: #006089;
    border: 3px solid #d9812f;
    border-radius: 5px;
    font-size: 16px;
    padding: 7px 10px; 
    transition: all 0.8s ease;</string>
   </property>
   <property name="text">
    <string>Завершить</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="createLogEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ffa852</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="createPassEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ffa852;
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendCodeEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ffa852</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
